src/routes/comments.ts

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import { getComments } from "../database/comment";
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    console.log("In Item...");
+    const comments = await getComments();
+
+    if (comments.length > 0) {
+      res.status(200).json(comments);
+    } else {
+      res.status(404).json({ message: "There are no available comments" });
+    }
+  } catch (error: any) {
```

Comment:
Let the variable be inferred as `unknown` to avoid accessing its properties. No need to retrieve a message here, unless there is a need to store the callstack in a logging system. Otherwise, log the whole error object.